client/functions.lua

```diff
@@ -382,6 +405,11 @@ function QBCore.Functions.GetPlate(vehicle)
     return QBCore.Shared.Trim(GetVehicleNumberPlateText(vehicle))
 end
 
+function QBCore.Functions.GetVehicleLabel(vehicle)
+    if vehicle == 0 or vehicle == nil then return end
```

Comment:
Turn these checks around because it might error now if the vehicle is nil

Reply:
Fixed :)